ethpm_types/contract_type.py

```diff
@@ -290,10 +313,21 @@ class ContractType(BaseModel):
     **NOTE**: This is not part of the canonical EIP-2678 spec.
     """
 
-    pcmap: Optional[dict] = None
+    pcmap: Optional[PCMap] = None
     """
     The program counter map representing which lines in the source code account for which
     instructions in the bytecode.
+
+    The compiler returns the information as string PC keys with list values. The values in the list
+    represent the starting line, starting column, ending line, and ending column of the relevant
+    statement in the source code.
+
+    Format Example::
+            {
+                "123": [1, 2, 1, 4],
```

Comment:
Here you say it's a string, but above it's an int

Reply:
It's an int when we return it (and we return a list of `PCMapItem` objects instead of a list of ints), but coming from the compiler it is a string

I'll remove this since really the compiler's format doesn't matter once we throw it into a Pydantic model

Updated in c795031 to clarify documentation and add some additional info on `PCMap` and `PCMap.parse()`. Also removed `pc` from `PCMapItem` since it shouldn't have been in there, was a leftover